client/package.json

```diff
@@ -4,7 +4,8 @@
   "private": true,
   "dependencies": {
     "bootstrap": "^4.0.0",
-    "node-sass-chokidar": "^1.0.1",
+    "node-sass-chokidar": "^1.1.0",
+    "npm": "^5.7.1",
```

Comment:
I looked it up (https://docs.npmjs.com/getting-started/installing-node). I think we need to remove this one, `npm` is not meant to be in `package.json`, not even as a developer dependency. 
In order to keep `package-lock.json` in sync with `package.json`, you'll need to do it with `npm uninstall` like this:
```
npm uninstall npm --save
```